src/main/java/jimmy/mcgymmy/model/util/SampleDataUtil.java

```diff
@@ -46,10 +46,13 @@ public static ReadOnlyMcGymmy getSampleMcGymmy() {
     /**
      * Returns a tag set containing the list of strings given.
      */
-    public static Set<Tag> getTagSet(String... strings) {
-        return Arrays.stream(strings)
-                .map(Tag::new)
-                .collect(Collectors.toSet());
+    public static Set<Tag> getTagSet(String... strings) throws IllegalValueException {
+        Set<Tag> collect = new HashSet<>();
+        for (String string : strings) {
+            Tag tag = new Tag(string);
+            collect.add(tag);
```

Comment:
Not sure if this matters for coding standard, but `collect` is most usually denoted as a verb, maybe change name of HashSet to `collectSet` or `collectionSet` or `tagCollection`?